indico/MaKaC/webinterface/pages/conferences.py

```diff
@@ -5812,27 +5813,20 @@ def _applyDecoration( self, body ):
         return "%s%s%s"%( self._getHeader(), body, self._getFooter() )
 
     def _getBody( self, params ):
-        params["URL2Back"] = urlHandlers.UHConfModifDisplay.getURL(self._conf)
-        params["cssurl"] = ""
-        params['selectedCSSId'] = ""
-        if self._selectedCSS:
-            params["cssurl"] = self._selectedCSS.getURL()
-            params['selectedCSSId'] = self._selectedCSS.getId()
-        elif self._styleMgr.getCSS():
-            params["cssurl"] = self._styleMgr.getCSS().getURL()
-            params['selectedCSSId'] = self._styleMgr.getCSS().getId()
-        params["saveCSS"]=urlHandlers.UHUseCSS.getURL(self._conf)
         params['confId'] = self._conf.getId()
-        params["previewURL"]= urlHandlers.UHConfModifPreviewCSS.getURL(self._conf)
-        params["templatesList"]=[]
-        if self._styleMgr.getLocalCSS():
-            params["templatesList"].append(self._styleMgr.getLocalCSS())
-        params["templatesList"].extend(self._cssTplsModule.getCssTplsList())
+        params['conf'] = self._conf
+
+        form = CSSSelectionForm(event=self._conf)
+        form.theme.data = request.values.get('theme')
```

Comment:
you immediately know that it's GET data and that you are not going to expect POST data